iaas-common/src/main/java/io/github/cloudiator/persistance/PricingModel.java

```diff
@@ -0,0 +1,203 @@
+package io.github.cloudiator.persistance;
+
+import io.github.cloudiator.domain.DiscoveryItemState;
+import org.apache.kafka.common.protocol.types.Field;
+
+import javax.annotation.Nullable;
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Inheritance(strategy = javax.persistence.InheritanceType.TABLE_PER_CLASS)
+public class PricingModel extends Model {
+    @Column(nullable = false, updatable = false)
+    @Lob
+    private String cloudUniqueId;
+    @Column(nullable = false, updatable = false)
+    private String providerId;
+    @Column(nullable = false, updatable = false)
+    private String name;
+    @Column(nullable = false)
+    private String instanceName;
+    @Column(nullable = false)
+    private String cloudServiceProviderName;
+    @Column(nullable = false)
+    private String currency;
+    @Column(nullable = true)
```

Comment:
All `@Column` annotations with only one attribute - `nullable = true` are redundant because `true` is default value for `nullable`.

Reply:
done